ansible-tailscale-inventory.py

```diff
@@ -47,4 +47,11 @@
         inventory[v["OS"]] = {"hosts": []}
     inventory[v["OS"]]["hosts"].append(v["HostName"])
 
+    if "Tags" in v:
+        for tag in v["Tags"]:
```

Comment:
This is a great idea 

One concern I have is that the group name ends up being formatted as `tag:mytag`, and ansible host patterns use `:` as the default separator. You _can_ use `,` as a separator but I worry this could trip up users, since a lot documentation out there will use `:` as a separator, and if you're just targeting `tag:mytag`, the pattern needs a `,` at the end to cause ansible not to consider the `:` as a separator (`tag:mytag,`).

What do you think about converting the `:` to an `_`?

Reply:
Good point, I've updated with the fix